backend/src/test/java/com/bakdata/conquery/util/dict/TreeTestUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.bakdata.conquery.util.dict;
+
+import java.util.Comparator;
+import java.util.IdentityHashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.UUID;
+
+public class TreeTestUtil {
+
+	public static <T> Comparator<T> shuffle(Random random) {
```

Comment:
Warum braucht man hier die UUID? und den Random?

Reply:
Ich hab das nicht geschrieben, aber es assigned einfach eine random UUID jedem Element und benutzt die als comparison Wert, damit das zufällig sortiert ist. Warum das in dem TernaryTree Test so gebraúcht wird weiß ich nicht

Reply:
Okay, ich denke man könnte auch einfach random ints/Integer zurückgeben